multistream/multistream.go

```diff
@@ -5,7 +5,7 @@ package multistream
 import (
 	"net"
 
-	mss "github.com/whyrusleeping/go-multistream"
+	mss "gx/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```

Comment:
@whyrusleeping this is fine with me but is not go-gettable. paths can be:

```
mss "ipfs.io/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```
no?

or similarly:
```
mss "gx.io/ipfs/Qmf91yhgRLo2dhhbc5zZ7TxjMaR1oxaWaoc9zRZdi1kU4a/go-multistream"
```